### ISSUE TITLE
Nuxt 4 support

### ISSUE THREAD

**Issue by PMLS3** [shuriken-ui/nuxt#127]
Thanks for great repo, is there a chance we can get nuxt4 support i.e. nightly?

**Comment by PMLS3** [comment]
![Uploading Screenshot 2024-07-30 at 06.34.26.png…]()

**Comment by stafyniaksacha** [comment]
Yes, this is our internal linear.<br><br>We have not planned it for now, we have to look up how layers are integrated in v4 and may be switch to a module instead<br><br>Here is a link to try current shuriken ui layer in nuxt in v4 compatibility mode<br>[https://stackblitz.com/edit/nuxt-starter-uh9uuh](https://stackblitz.com/edit/nuxt-starter-uh9uuh)

**Comment by PMLS3** [comment]
Appreciate
